priv/static/index.html

```diff
@@ -7,12 +7,23 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link href="css/style.css" rel="stylesheet">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+    crossorigin="anonymous">
 </head>
 
 <body>
-  <div id="labels">
+  <div id="landingLabelsContainer">
+    <div id="landingLabels">
+      <input type="text" id="siteSearch" class="input-lg" placeholder="Enter a website you love. Discover more like it." autofocus>
+      <button type="button" id="exploreButton" class="btn-primary btn-lg btn-huge">Explore!</button>
+    </div>
+  </div>
+
+  <div id="mapLabels" hidden=false>
+    <img src="https://image.flaticon.com/icons/svg/126/126492.svg" id="backButton">
```

Comment:
I don't like relying on fetching images from external sites long term. Also is this licensed for our use?